spectree/utils.py

```diff
@@ -76,7 +76,14 @@ def parse_comments(func: Callable[..., Any]) -> Tuple[Optional[str], Optional[st
     return summary, description
 
 
-def parse_request(func: Any) -> Dict[str, Any]:
+def has_examples(schema_exta: dict) -> bool:
+    for _, v in schema_exta.items():
+        if isinstance(v, dict) and "value" in v.keys():
```

Comment:
nitpick: `.keys()` is superfluous since it's the default place where `in` is checking

```suggestion
        if isinstance(v, dict) and "value" in v:
```